hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Filter/ReviewOrgUnitChangesFilter.tsx

```diff
@@ -117,15 +226,70 @@ export const ReviewOrgUnitChangesFilter: FunctionComponent<Props> = ({
                     options={groupOptions}
                     loading={isLoadingGroups}
                     labelString={formatMessage(MESSAGES.group)}
-                    disabled
-                    helperText={formatMessage(MESSAGES.featureDisabled)}
                 />
+                <Box mt={2}>
+                    {!showAdvancedSettings && (
+                        <Typography
+                            data-test="advanced-settings"
+                            variant="overline"
+                            sx={styles.advancedSettings}
+                            onClick={() => setShowAdvancedSettings(true)}
+                        >
+                            {formatMessage(MESSAGES.showAdvancedSettings)}
+                        </Typography>
+                    )}
+                    {showAdvancedSettings && (
+                        <>
+                            <InputComponent
+                                type="select"
+                                disabled={isFetchingDataSources}
+                                keyValue="source"
+                                onChange={handleDataSourceVersionChange}
+                                value={isFetchingDataSources ? '' : dataSource}
+                                label={MESSAGES.source}
+                                options={dataSources}
+                                loading={isFetchingDataSources}
+                            />
```

Comment:
Could you move this one outside of the advanced settings, so it's the same as in Org Units?